src/components/form/form.d.ts

```diff
@@ -28,6 +29,9 @@ export interface FormProps {
 
  /** The total number of warnings present in the form */
  warningCount?: number;
+
+ /** Multipler for vertical spacing between fields, times theme.spacing */
+ fieldSpacingMultiplier?: FormSpacingMultiplier;
```

Comment:
How about this description and prop name? : 

/** Spacing between form fields, given number will be multiplied by base spacing unit (8) */
 fieldSpacing?: FormFieldSpacing;